app/views/users/edit.html.erb

```diff
@@ -1,10 +1,12 @@
 <h1>Editing user</h1>
 
-<%= render "form", user: @user %>
+<% if current_user %>
+  <%= render "form", user: @user %>
+<% end %>
 
 <br>
 
-<div>
+<%# <div>
   <%= link_to "Show this user", @user %> |
-  <%= link_to "Back to users", users_path %>
-</div>
+  <%# <%= link_to "Back to users", users_path %>
```

Comment:
do not comment just remove if not needed.

Reply:
done